packages/dotcom-build-image-uploader/README.md

```diff
@@ -0,0 +1,62 @@
+# @financial-times/dotcom-build-image-uploader
+
+This package exports a Webpack plugin to provide support for static image assets in our asset pipeline.
+The plugin will copy all image files contained in the `client/` directory, and any of its subdirectories, into the configured public directory. 
+
+
+## Requirements
+
+* Node 12.x
+
+
+## Installation
+
+```sh
+npm install --save-dev @financial-times/dotcom-build-image-uploader
+```
+
+## Usage
+
+After installing the package you must add it to the list of plugins in your project's `webpack.config.js` configuration file:
+
+```diff
++ const { PageKitImageUploaderPlugin } = require('@financial-times/dotcom-build-image-uploader')
+
+module.export = {
+  plugins: [
++    new PageKitImageUploaderPlugin()
+  ]
+}
+```
+
+## Arguments
+
+- `basePath` - Directory to look in for images. Defaults to `./client/`.
```

Comment:
The other `dotcom-build-` package readmes all follow the same format, I would recommend copying one of them and amending as needed for consistency and to ensure nothing is missed,